osf/models/files.py

```diff
@@ -672,6 +672,14 @@ def restore(self, recursive=True, parent=None, save=True, deleted_on=None):
         return tf
 
 
+class FileVersionUserMetadata(BaseModel):
+    user = models.ForeignKey('OSFUser')
```

Comment:
Drive-by: need to specify `on_delete` for `ForeignKey` so we don't get [deprecation warnings]( https://docs.djangoproject.com/en/2.0/internals/deprecation/#deprecation-removed-in-2-0).

Reply:
@erinspace, on second thought, I don't think this model is necessary. Django [creates a through table for you](https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.ManyToManyField.through) when you define a M2M (`seen_by` on `FileVersion`). You would only define your own intermediary table when there's additional data to store.

Reply:
It's not strictly necessary to have the through table, but it's more explicit to have it and also `created` and `modified` fields, which can be useful metadata for debugging.

Furthermore, explicitly defining the through table is more future-proof--it's easier to add it now than it would be in the future (after rows have been added to the implicit table).